controllers/telemetry/logpipeline_controller_test.go

```diff
@@ -466,42 +468,83 @@ var _ = Describe("LogPipeline controller", Ordered, func() {
 		})
 
-		It("Creates a logpipeline with missing secret reference", func() {
+		It("Creates a LogPipeline with missing secret reference", func() {
 			Expect(k8sClient.Create(ctx, missingSecretRefLogPipeline)).Should(Succeed())
 		})
 
-		It("Should create a fluent bit configmap which contains healthy pipeline", func() {
+		It("Should create a fluent-bit-sections configmap which contains the healthy pipeline", func() {
 			Eventually(func() error {
 				return validateKeyExistsInFluentbitSectionsConf(ctx, "healthy-logpipeline.conf")
 			}, timeout, interval).Should(BeNil())
 		})
 
-		It("Should not have a unhealthy pipeline in fluent-bit configmap", func() {
+		It("Should not include the LogPipeline with missing secret in fluent-bit-sections configmap", func() {
 			expectedErr := errors.New("did not find the key: missing-secret-ref-logpipeline.conf")
-			Eventually(func() error {
+			Consistently(func() error {
 				return validateKeyExistsInFluentbitSectionsConf(ctx, "missing-secret-ref-logpipeline.conf")
 			}, timeout, interval).Should(Equal(expectedErr))
 		})
 
-		It("Should update config when secret is created", func() {
+		It("Should update fluent-bit-sections configmap when secret is created", func() {
 			Expect(k8sClient.Create(ctx, pipelineSecret)).Should(Succeed())
 			Eventually(func() error {
 				return validateKeyExistsInFluentbitSectionsConf(ctx, "missing-secret-ref-logpipeline.conf")
 			}, timeout, interval).Should(BeNil())
 		})
 
-		It("Should remove logpipeline from configmap when secret is deleted", func() {
+		It("Should remove LogPipeline from configmap when secret is deleted", func() {
 			Expect(k8sClient.Delete(ctx, pipelineSecret)).Should(Succeed())
 			Eventually(func() error {
 				return validateKeyExistsInFluentbitSectionsConf(ctx, "missing-secret-ref-logpipeline.conf")
 			}, timeout, interval).Should(BeNil())
 		})
 
 	})
+
+	Context("When creating a LogPipeline with Loki Output", Ordered, func() {
+		pipelineName := "pipeline-with-loki-output"
+		pipelineWithLokiOutput := &telemetryv1alpha1.LogPipeline{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: pipelineName,
+			},
+			Spec: telemetryv1alpha1.LogPipelineSpec{
+				Output: telemetryv1alpha1.Output{
+					Loki: &telemetryv1alpha1.LokiOutput{
+						URL: telemetryv1alpha1.ValueType{
+							Value: "http://logging-loki:3100/loki/api/v1/push",
+						},
+					},
+				}},
+		}
+
+		BeforeAll(func() {
+			Expect(k8sClient.Create(ctx, pipelineWithLokiOutput)).Should(Succeed())
+			DeferCleanup(func() {
+				Expect(k8sClient.Delete(ctx, pipelineWithLokiOutput)).Should(Succeed())
+			})
+		})
+
+		It("Should not include the LogPipeline in fluent-bit-sections configmap", func() {
+			expectedErr := errors.New("did not find the key: pipeline-with-loki-output.conf")
+			Consistently(func() error {
+				return validateKeyExistsInFluentbitSectionsConf(ctx, "pipeline-with-loki-output.conf")
+			}, timeout, interval).Should(Equal(expectedErr))
```

Comment:
Let's use a different timeout for `Consistently`. This will poll for 1 minutes. Check the timeouts in the e2e tests.

Reply:
The timeout is actually set as 10 seconds https://github.com/kyma-project/telemetry-manager/blob/b4e48fe8692b3fbad9aa08ab03aa7b1456021d5a/controllers/telemetry/logpipeline_controller_test.go#L62